app-misc/pwsafe/pwsafe-0.2.0-r5.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A Password Safe compatible command-line password manager"
+HOMEPAGE="http://nsd.dyndns.org/pwsafe/"
+SRC_URI="http://nsd.dyndns.org/pwsafe/releases/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="X libressl readline"
+
+DEPEND="sys-libs/ncurses:0=
+	!libressl? ( dev-libs/openssl:0= )
+	libressl? ( dev-libs/libressl:0= )
+	readline? ( sys-libs/readline:0= )
+	X? ( x11-libs/libSM
+		x11-libs/libICE
+		x11-libs/libXmu
+		x11-libs/libX11 )"
+RDEPEND="${DEPEND}
+	!app-admin/passwordsafe"
+
+src_prepare() {
+	eapply -p0 "${FILESDIR}/${P}-cvs-1.57.patch"
```

Comment:
Nowadays (with EAPI 7) we should be able to just define PATCHES=() as an array above src_prepare. -p0 shouldn't be needed but you may need to adjust the patch filenames in the patches (easily done) to make them apply automatically.

Let me know if you need any help.

I'd also then drop the src_prepare definition as it shouldn't be needed.

Reply:
I thought it better not to modify the patch files that were already there, but if you want then of course that will work. Done.